lib/tyrant/railtie.rb

```diff
@@ -1,17 +1,18 @@
 module Tyrant
   class Railtie < Rails::Railtie
-      require "warden"
+    require "warden"
 
-      # DISCUSS: it will be configurable what user class etc. and might be moved to Ops.
-      # DISCUSS: this should actually use Tyrant::Session to log in and out user?
-      config.app_middleware.use Warden::Manager do |config|
-        Warden::Manager.serialize_into_session do |user|
-          user.id
-        end
+    # DISCUSS: this should actually use Tyrant::Session to log in and out user?
+    config.app_middleware.use Warden::Manager do |config|
```

Comment:
And specify it is not the same as the current record you want to serialize class? I mean, if you serialize something, you want to get the same thing and not search inside another type of class...